terraform/main.tf

```diff
@@ -44,12 +45,38 @@ data "aws_iam_policy_document" "s3" {
 
 module "scheduled_ingest" {
   source              = "github.com/terraform-community-modules/tf_aws_lambda_scheduled"
-  lambda_name         = "inclucivics_ingest"
+  lambda_name         = "${var.lambda_name}"
   runtime             = "nodejs6.10"
   lambda_zipfile      = "ingest.zip"
   source_code_hash    = "${data.archive_file.ingest.output_base64sha256}"
   handler             = "index.handler"
   schedule_expression = "rate(1 day)"
-  timeout             = 200
+  timeout             = 60
   iam_policy_document = "${data.aws_iam_policy_document.s3.json}"
 }
+
+resource "aws_cloudwatch_metric_alarm" "failure" {
+  alarm_name          = "inclucivics_ingest_failure"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = "1"
+  metric_name         = "Errors"
+  namespace           = "AWS/Lambda"
+  period              = 300
+  threshold           = 0
+  statistic           = "SampleCount"
+
+  dimensions {
+    FunctionName = "${var.lambda_name}"
+  }
+
+  alarm_description = "Send an email when the lambda function ends in an Error"
+  alarm_actions     = ["${module.email.arn}"]
+}
+
+module "email" {
+  source              = "github.com/deanwilson/tf_sns_email"
```

Comment:
This is weird - who's this?

Reply:
Just a random pair of folks who made a terraform module in 2017.  I'd like to pin it to a version so it can't be augmented so easily, but not sure how with this one.